cernopendata_client/downloader.py

```diff
@@ -28,6 +30,32 @@
 
 from .validator import validate_range
 from .printer import display_message
+from .verifier import get_file_checksum
+from .config import DOWNLOAD_RETRY_SLEEP_TIME
+
+
+def check_error(path=None, file_location=None, protocol=None, retry=None):
+    """Check downloaded file for error page."""
+    file_name = file_location.split("/")[-1]
+    file_path = path + "/" + file_name
+    error_page = (
+        os.path.getsize(file_path) == 3846
+        and get_file_checksum(file_path) == "adler32:a82d5324"
+    )
+    if error_page:
+        for _retry in range(0, retry + 1):
+            if _retry == retry:
+                display_message(msg_type="error", msg="Number of retries exceeded.")
+                sys.exit(1)
+            display_message(
+                msg_type="info", msg="Retrying {}/{}".format(_retry + 1, retry)
```

Comment:
Instead of `info` it would be better to use `note`, because we are retrying the same download... so it is rather secondary header instead of primary header when we display things:

```
==> Downloading file 1 of 1
  -> File: ./3005/0d0714743f0204ed3c0144941e6ce248.configFile.py
  -> Progress: 3/3 kiB (100%)
==> Verifying file 0d0714743f0204ed3c0144941e6ce248.configFile.py...
  -> Expected size 3644, found 3644
  -> Expected checksum adler32:be83a186, found adler32:be83a186
==> Success!
==> Downloading file 1 of 1
  -> File: ./3005/0d0714743f0204ed3c0144941e6ce248.configFile.py
  -> Progress: 3/3 kiB (100%)
==> Retrying 1/10
  -> File: ./3005/0d0714743f0204ed3c0144941e6ce248.configFile.py
  -> Progress: 3/3 kiB (100%)
==> Retrying 2/10
```